docs: New documentation (#858)
dbbs-lab/bsb-core: docs/placement/placement-strategies.rst

*Class*: :class:`bsb.placement.RandomPlacement <.placement.random.RandomPlacement>`

> This class place cells in random postition without caring about overlaps. Here is an example with 10 cells.

Reviewer: GPT
```suggestion
This class places cells in random postitions without considering any overlap. This is an example with 10 cells:
```

.. code-block:: json

  {
    "cell_types": {
      "golgi_cell": {
        "placement": {
          "class": "bsb.placement.particle.RandomPlacement",
          "layer": "granular_layer",
          "count": 10
          }
      },
    }

ParallelArrayPlacement
**********************

*Class*: :class:`bsb.placement.ParallelArrayPlacement
<.placement.arrays.ParallelArrayPlacement>`

This class place cells in an aligned array, it create a lattice with fixed spacing and with the desired angle.

> It is necessary to specify ``spacing_x`` and ``angle`` attributes.

Reviewer: GPT
